man/docker-network-create.1.md

```diff
@@ -0,0 +1,55 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-create - create a new network
+
+# SYNOPSIS
+**docker network create**
+
+[**-d**|**--driver**=[**name**]]     
+[**--help**]
+
+# DESCRIPTION
+
+Creates a new network. Driver accepts any of the following `name` values:
+
+
+| Value     | Description                                                                                                                                                                         |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bridge`  | Uses Linux bridging and iptables to connect containers. This creates a single bridge, called `docker0` by default, and attaches a veth pair between the bridge and every container. |
+| `null`    | A noop implementation of the driver API, used only in cases where no networking is desired. This is to provide backward compatibility to the Docker's --net=none option.            |
```

Comment:
```
$ docker-1.9.0-rc1 $(dm config rc1) network create -d null foobar
Error response from daemon: only one instance of "null" network is allowed

$ docker-1.9.0-rc1 $(dm config rc1) network create -d host foobar
Error response from daemon: only one instance of "host" network is allowed
```

It seems pretty pointless to include these in the UI if they aren't usable. I'm also wondering if they provide any value being present in `network ls` as it could be pretty confusing to a new user. WDYT?
/cc @madhu @bfirsh